src/IW/Core/Issue.hs

```diff
@@ -8,12 +8,13 @@ module IW.Core.Issue
   
 import IW.Core.SqlArray (SqlArray (..))
 import IW.Core.Repo (RepoName (..), RepoOwner (..))
+import IW.Core.Url (Url (..))
 
 
 -- | Wrapper for issue label names.
 newtype Label = Label { unLabel :: Text }
-    deriving stock   (Generic, Show)
-    deriving newtype (Eq, Ord, FromField, ToField, FromJSON, ToJSON, FromHttpApiData)
+    deriving stock   (Generic)
+    deriving newtype (Eq, Ord, Show, FromField, ToField, FromJSON, ToJSON, FromHttpApiData)
```

Comment:
I'm not sure whether @vrom911 meant to change every `Show` deriving or just for `Url`... :thinking:

Reply:
@chshersh probably just URL